connector/profiler/README.md

```diff
@@ -54,7 +54,7 @@ Then enable the profiling in the configuration.
   <td><code>spark.executor.profiling.dfsDir</code></td>
   <td>(none)</td>
   <td>
-      An HDFS compatible path to which the profiler's output files are copied. The output files will be written as <i>dfsDir/application_id/profile-appname-exec-executor_id.jfr</i> <br/>
+      An HDFS compatible path to which the profiler's output files are copied. The output files will be written as <i>dfsDir/{{APP_ID}}/profile-{{APP_ID}}-exec-{{EXECUTOR_ID}}.jfr</i> <br/>
```

Comment:
actually, `{{APP_ID}}` is `nameForAppAndAttempt`(see code) for YARN cluster mode, but writing behavior details in the configuration description is a little bit verbose ...

please let me know if you have better suggestion to polish this sentence.

Reply:
I think this is fine.